examples/zig-basic/src/basic.zig

```diff
@@ -0,0 +1,57 @@
+const std = @import("std");
+const worker = @import("worker");
+
+var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
+const allocator = arena.allocator();
+
+fn requestFn(resp: *worker.Response, r: *worker.Request) void {
+
+    var payload: []const u8 = "";
+    var reqBody = r.data;
```

Comment:
This example is not working after the latest commits. It was a key change:

```suggestion
    var reqBody = r.body;
```

Reply:
Good catch, thanks for testing this!